packages/commercetools/api-client/src/api/deleteCart/defaultMutation.ts

```diff
@@ -0,0 +1,12 @@
+import gql from 'graphql-tag';
+import { CartFragment } from '../../fragments';
+
+export default gql`
+  ${CartFragment}
+
+  mutation deleteCart($id: String!, $version: Long!, $locale: Locale!, $acceptLanguage: [Locale!]) {
```

Comment:
I think the cart here will get its status set to `ordered` and wont be queried as an `active cart` anymore via getMe endpoint